src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -1090,6 +1090,11 @@
     <Field type="boolean" name="onOutOfMemoryError" label="On Out of Memory Error" />
   </Event>
 
+  <Event name="GCLocker" category="Java Virtual Machine, GC, Detailed" description="GC Locker Information" label="GC Locker" startTime="true" thread="true" stackTrace="true">
+    <Field type="uint" name="lockCount" label="Lock Count" />
+    <Field type="uint" name="stallCount" label="Stall Count" />
```

Comment:
Please add descriptions to the fields as mentioned above.

Reply:
added